charts/rstudio-workbench/README.md.gotmpl

```diff
@@ -16,7 +16,7 @@ This chart requires the following in order to function:
   * If `homeStorage.create` is set, a PVC that relies on the default storage class will be created to generate the PersistentVolume. 
     Most Kubernetes environments do not have a default storage class that you can use with `ReadWriteMany` access mode out-of-the-box. 
     In this case, we recommend you disable `homeStorage.create` and  create your own `PersistentVolume` and `PersistentVolumeClaim`, then mount them 
-    into the container by specifying the `pod.volumes` and `pod.volumeMounts` parameters.
+    into the container by specifying the `pod.volumes` and `pod.volumeMounts` parameters, or by specifying your `PersistentVolumeClaim` using `sharedStorage.name` and `sharedStorage.mount`.
```

Comment:
```suggestion
    into the container by specifying the `pod.volumes` and `pod.volumeMounts` parameters, or by specifying your `PersistentVolumeClaim` using `homeStorage.name` and `homeStorage.mount`.
```

Sorry. my original suggestion mixed the wrong configs up (since we were talking about home storage but there wasn't yet the mechanism you just added for it).